src/pages/index.astro

```diff
@@ -1,33 +1,125 @@
 ---
 import MainLayout from '../layouts/MainLayout.astro';
+import {COMMUNITY_INVITE_URL, GITHUB_ORG_URL, REDDIT_URL} from '../consts';
 ---
 
 <MainLayout title="" }>
-	<div class="logo">
-		<img src="/syscraft-logo.gif" alt="Syscraft logo"/>
-		Syscraft
-	</div>
-</MainLayout>
+    <div class="logo">
+        <img src="/syscraft-logo.gif" alt="Syscraft logo"/>
+        Syscraft
+    </div>
+
+    <h1>Welcome</h1>
+    <p>
+        Syscraft is a community for everyone involved in the Minecraft server
+        community to gather, ask for help, share stories, and make the
+        server world a better place to hang out!
+    </p>
+    <p>
+        You're welcome as an admin, moderator, developer,
+        owner, host, or even as a player interested in how it all works!
+    </p>
+    <hr/>
+
+    <h1>Navigating</h1>
+    <p>
+        On this site, you can find useful guides and resources on creating,
+        maintaining and growing Minecraft servers and their communities!
+    </p>
+    <p>
+        If you are new to server hosting, we'd recommend skimming through each guide on the sidebar.
+    </p>
+    <p>
+        If you are looking for a specific page, use the <code><kbd>CTRL</kbd>+<kbd>K</kbd></code>
```

Comment:
For Mac users like me, it's `CMD + K` - either we add that here too or just remove it (the user could see there is a shortcut by looking at the search box and figure it out for themselves).

Reply:
Fair point, I just removed it 👍